webextensions/manifest.json

```diff
@@ -31,26 +31,21 @@
   },
   "browser_action": {
     "default_title": "__MSG_sidebarTitle__",
-    "default_icon": {
-      "16": "resources/16x16.svg",
-      "20": "resources/20x20.svg",
-      "24": "resources/24x24.svg"
-    },
```

Comment:
These lines were intentionally left for the future. I'm planning to remove "theme_icons" after ESR60 become outdated by ESR68. The reason why I left them here is: I cannot put any comment in a JSON and I'll forget the plan.

Anyway, then I'll remove "-light" suffixes from "default_icon" under "sidebar_actions" and I'll copy those lines to "browser_actions".